src/main/java/com/um/inventoryservice/BusinessLayer/DataSetupService.java

```diff
@@ -29,16 +29,47 @@ public void run(String... args) throws Exception {
         OrderDTO o1 = new OrderDTO("1", new ArrayList<>());
         OrderDTO o2 = new OrderDTO("2", new ArrayList<>());
         OrderDTO o3 = new OrderDTO("2", new ArrayList<>());
+        OrderDTO o4 = new OrderDTO("3", new ArrayList<>());
+        OrderDTO o5 = new OrderDTO("4", new ArrayList<>());
+        OrderDTO o6 = new OrderDTO("5", new ArrayList<>());
+        OrderDTO o7 = new OrderDTO("6", new ArrayList<>());
+        OrderDTO o8 = new OrderDTO("7", new ArrayList<>());
+        OrderDTO o9 = new OrderDTO("5", new ArrayList<>());
+        OrderDTO o10 = new OrderDTO("5", new ArrayList<>());
         StockOrderDTO so1 = new StockOrderDTO("2454544", "this is the test for stock item 1", 6);
         StockOrderDTO so2 = new StockOrderDTO("7486504", "this is the test for stock item 2", 7);
         StockOrderDTO so3 = new StockOrderDTO("9735693", "this is the test for stock item 3", 8);
         StockOrderDTO so4 = new StockOrderDTO("7486504", "this is the test for stock item 3 x2", 3);
+        StockOrderDTO so5 = new StockOrderDTO("2454544", "this is the test for stock item 5", 15);
```

Comment:
we can now put data that will represent real-life situations for the system.

Reply:
Will update the sample data at next opportunity.